packages/taquito-local-forging/test/taquito-local-forging.spec.ts

```diff
@@ -19,153 +20,255 @@ describe('Forge and parse operations default protocol', () => {
+
+      const consumer = Uint8ArrayConsumer.fromHexString(hexToParse)
+      expect(decoders[codec](consumer)).toStrictEqual(gotCodec)
+      expect((schemaDecoder(decoders)(SeedNonceRevelationSchema)(consumer))).toBeDefined
+      done();
+    });
+
+    test(`Verify Arrow Functions for CODEC.SECRET is toHexString(val.consume(20))`, async (done) => {
+      const codec = CODEC.SECRET
+      const myGetCodec = getCodec(codec, ProtocolsHash.PtKathman)
+      const encodeCodec = myGetCodec.encoder('0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a')
+      expect(encodeCodec).toEqual("0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a")
+      const decodeCodec = myGetCodec.decoder('0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a')
+      expect(decodeCodec).toEqual("0572cbea904d67468808c8eb50a9450c9721db30")
+      done();
+    });
+
+    test(`Verify Arrow Function for CODEC.RAW is toHexString(val.consume(32)),`, async (done) => {
+      const codec = CODEC.RAW
+      const myGetCodec = getCodec(codec, ProtocolsHash.PtKathman)
+      const encodeCodec = myGetCodec.encoder('0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a')
+      expect(encodeCodec).toEqual("0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a")
+      const decodeCodec = myGetCodec.decoder('0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a')
+      expect(decodeCodec).toEqual("0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a")
+      done();
+    });
+
+    test(`Verify Arrow Function for CODEC.OP_ACTIVATE_ACCOUNT`, async (done) => {
+      const codec = CODEC.OP_ACTIVATE_ACCOUNT
+      const myGetCodec = getCodec(codec, ProtocolsHash.PtKathman)
+      const gotCodec = myGetCodec.decoder('0572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62a')
+      expect(gotCodec).toEqual({ "pkh": "tz1L8qbHKzqFmLiZqxaJdMEWrCwmmrSw2wj4", "secret": "9128012543902d0ac358a62a" })
+
+      const consumer = Uint8ArrayConsumer.fromHexString(hexToParse)
+      expect(decoders[codec](consumer)).toStrictEqual({ "pkh": "tz1L8qbHKzqFmLiZqxaJdMEWrCwmmrSw2wj4", "secret": "9128012543902d0ac358a62ae28f75bb8f1c7c42" })
+      expect((schemaDecoder(decoders)(ActivationSchema)(consumer))).toBeDefined
+      expect((schemaDecoder(decoders)(ActivationSchema)(consumer))).toStrictEqual({ "pkh": "8wkhLGYvWfaqQXYRo5yCvSe4mVr314j", "secret": "" })
+      done();
+    });
+
+    test(`test getCodec to verify codec Manager sent to decoders is defined`, async (done) => {
+      const myGetCodec = getCodec(CODEC.MANAGER, ProtocolsHash.PtKathman).decoder('7c842c15c8b0c8fd228e6cb5302a50201f41642dd36b699003fb3c857920bc9d')
+      expect(myGetCodec).toEqual({ "branch": "BLf7wKNryZRXibjzM4TjNBSMgNN4qJVhxRRuxo3uu3SegsFqkUd", "contents": [] })
+      done();
+    });
+
+    test(`test getCodec to verify codec secret sent to decoders is defined`, async (done) => {
```

Comment:
Kind if duplicate of `Verify Arrow Functions for CODEC.SECRET is toHexString(val.consume(20)`

Reply:
True. Deleted the dupe.